2015-09-14 09:39:16 [ main:31698 ] - [ ERROR ] 执行sql='SELECT  PackageGUID AS packageGuid,osi.FastFlow AS fastflowName,  Gesamtpreis AS gesamtpreis,tb.Currency AS currency,  tb.OrderGUID AS orderGuid,tb.Code AS productCode,qty=1,  tb.Referenznummer AS referenznummer,tb.SingleGuid AS singleGuid  ,osi.State,osi.PostDate,osi.CreateTime ,osi.TrackingNumber,osi.StockCode,osi.Barcode  FROM  dbo.tborder_sendinfo osi WITH(NOLOCK)  INNER JOIN dbo.tbOrder tb WITH(NOLOCK) ON osi.OrderID = tb.OrderID  WHERE osi.PostDate>=? AND osi.PostDate<=?'失败，错误原因：查询超时。'
参数：param1=2015-09-13 00:00:00;param2=2015-09-13 23:59:59;

com.microsoft.sqlserver.jdbc.SQLServerException: 查询超时。
	at com.microsoft.sqlserver.jdbc.TDSCommand.checkForInterrupt(IOBuffer.java:5918)
	at com.microsoft.sqlserver.jdbc.TDSParser.parse(tdsparser.java:70)
	at com.microsoft.sqlserver.jdbc.SQLServerResultSet.<init>(SQLServerResultSet.java:311)
	at com.microsoft.sqlserver.jdbc.SQLServerStatement.getNextResult(SQLServerStatement.java:1526)
	at com.microsoft.sqlserver.jdbc.SQLServerPreparedStatement.doExecutePreparedStatement(SQLServerPreparedStatement.java:404)
	at com.microsoft.sqlserver.jdbc.SQLServerPreparedStatement$PrepStmtExecCmd.doExecute(SQLServerPreparedStatement.java:350)
	at com.microsoft.sqlserver.jdbc.TDSCommand.execute(IOBuffer.java:5696)
	at com.microsoft.sqlserver.jdbc.SQLServerConnection.executeCommand(SQLServerConnection.java:1715)
	at com.microsoft.sqlserver.jdbc.SQLServerStatement.executeCommand(SQLServerStatement.java:180)
	at com.microsoft.sqlserver.jdbc.SQLServerStatement.executeStatement(SQLServerStatement.java:155)
	at com.microsoft.sqlserver.jdbc.SQLServerPreparedStatement.executeQuery(SQLServerPreparedStatement.java:285)
	at cn.hofan.spat.db.dal.sql.select.MyDaoSqlSelectHandler.findList2(MyDaoSqlSelectHandler.java:75)
	at cn.hofan.spat.db.dal.sql.select.MyDaoSqlSelectHandler.findList(MyDaoSqlSelectHandler.java:56)
	at cn.hofan.spat.db.dal.sql.select.SQLQueryImpl.list(SQLQueryImpl.java:95)
	at cn.hofan.ols.dao.common.impl.PackageProcessDaoImpl.IGetPackageAboardData(PackageProcessDaoImpl.java:183)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at cn.hofan.spat.aop.AOPInvocationHandler.invoke(AOPInvocationHandler.java:48)
	at com.sun.proxy.$Proxy4.IGetPackageAboardData(Unknown Source)
	at cn.hofan.ols.components.common.PackageProcessServiceImpl.IGetPackageProcessData(PackageProcessServiceImpl.java:267)
	at cn.hofan.ols.components.common.PackageProcessServiceImpl.IGetAndInsertPackageProcessData(PackageProcessServiceImpl.java:477)
	at cn.hofan.ols.system.impl.OrderProcessTest.PackageProcessServiceTest(OrderProcessTest.java:323)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:46)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:43)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:270)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:49)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:62)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:52)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:307)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)
2015-09-14 09:39:16 [ main:31702 ] - [ ERROR ] null,获取所有包裹时出现网络异常
